trussworks/react-uswds
# docs(example-app): Example app UI #202

# Summary

Bootstraps basic UI for the example application, including routing with [React Router](https://reacttraining.com/react-router/web/guides/quick-start).

## Related Issues or PRs

Closes #202 

## How To Test

Navigate to the `/example` directory, install dependencies (`yarn install`), and start the app (`yarn start`). Verify you can view the app, which currently just has two pages that are mostly empty (home & examples).

### Screenshots (optional)

![image](https://user-images.githubusercontent.com/2723066/83919112-93a40680-a73f-11ea-9cd3-587ecd3325a0.png)
